arbor/include/arbor/cable_cell.hpp

```diff
@@ -29,51 +29,153 @@ struct lid_range {
+// Sample value type: `cell_sample_range`
+// Sample metadata type: `mlocation`
+struct cell_probe_total_ion_current_density {
+    mlocation location;
+};
+
+// Total ionic current [nA] across membrance _excluding_ capacitive current across components of the cell.
+// Sample value type: `cell_sample_range`
+// Sample metadata type: `mcable_list`
+struct cell_probe_total_ion_current_cell {};
+
+// Total membrance current [nA] across components of the cell.
+// Sample value type: `cell_sample_range`
+// Sample metadata type: `mcable_list`
+struct cell_probe_total_current_cell {};
+
 // Value of state variable `state` in density mechanism `mechanism` in CV at `location`.
+// Sample value type: `double`
+// Sample metadata type: `mlocation`
 struct cell_probe_density_state {
     mlocation location;
     std::string mechanism;
     std::string state;
 };
 
+// Value of state variable `state` in density mechanism `mechanism` across components of the cell.
+// Sample value type: `cell_sample_range`
+// Sample metadata type: `mcable_list`
+struct cell_probe_density_state_cell {
+    std::string mechanism;
+    std::string state;
+};
+
 // Value of state variable `key` in point mechanism `source` at target `target`.
+// Sample value type: `double`
+// Sample metadata type: `cell_probe_point_info`
 struct cell_probe_point_state {
     cell_lid_type target;
     std::string mechanism;
     std::string state;
 };
 
+// Value of state variable `key` in point mechanism `source` at every target with this mechanism.
+// Metadata has one entry of type cell_point_info for each matched (possibly coalesced) instance.
+// Sample value type: `cell_sample_range`
+// Sample metadata type: `std::vector<cell_point_info>`
+struct cell_probe_point_state_cell {
+    std::string mechanism;
+    std::string state;
+};
+
 // Current density [A/m²] across membrane attributed to the ion `source` at `location`.
+// Sample value type: `double`
+// Sample metadata type: `mlocation`
 struct cell_probe_ion_current_density {
     mlocation location;
     std::string ion;
 };
 
-// Ionic internal concentration [mmol/L] of ion `source` at `location`, possibly interpolated.
+// Total ionic current [nA] attributed to the ion `source` across components of the cell.
+// Sample value type: `cell_sample_range`
+// Sample metadata type: `mcable_list`
+struct cell_probe_ion_current_cell {
+    std::string ion;
+};
+
+// Ionic internal concentration [mmol/L] of ion `source` at `location`.
+// Sample value type: `double`
+// Sample metadata type: `mlocation`
 struct cell_probe_ion_int_concentration {
     mlocation location;
     std::string ion;
 };
 
-// Ionic external concentration [mmol/L] of ion `source` at `location`, possibly interpolated.
+// Ionic internal concentration [mmol/L] of ion `source` across components of the cell.
+// Sample value type: `cell_sample_range`
+// Sample metadata type: `mcable_list`
+struct cell_probe_ion_int_concentration_cell {
```

Comment:
These `cell_X_cell` types are a little difficult to parse.

`cable_X_cell` and `cable_X` to indicate that we are talking about a cable cell specifically?

Reply:
Happy to rename `cell_probe_FOO` to `cable_probe_FOO`.